src/Components/Post.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import User from './User';
+import CommentList from './CommentList';
+
+const Post = ({ post }) => {
+  const { id, title, body, user, comment } = post;
+
+  return (
+
+    <div className="post">
+      <h1>
+ID:
+        {id}
+      </h1>
+      <h3>
+TITLE:
+        {title}
+      </h3>
+      <h4>
+POST:
+        {body}
+      </h4>
+      <User user={user} />
+      <CommentList comment={comment} />
```

Comment:
Ты передаешь туда массив из 5 комментариев, а не один комментарий
```suggestion
      <CommentList comments={comments} />
```

Reply:
+